testutil/ciao-down/cloudinit.go

```diff
@@ -55,6 +55,7 @@ write_files:
 
 runcmd:
  - echo "127.0.0.1 singlevm" >> /etc/hosts
```

Comment:
Oooh my earlier comment is irrelevant if the /etc/hosts is only modified inside the VM.  I'd still like a unique hostname.